README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
The delay could be cause due to a mismatch between the client and server machine internal clocks, but I'm not entirely sure.  Nevertheless, the **joint_trajectory_action** node is tightly couple to the trajectory execution on the robot (simulated or real) and it makes sense to have it run in the same machine as the rest of the robot nodes.  As that structure appears to be unattainable at the moment, for now lets keep the joint_trajectory_action on the server and eat up the 40 second delay penalty.
I'd like to get some basic functionality merged soon as we've been working on this PR for quite a while already. 
Down the road, I think the right solution to this connection sync issue would entail creating a "process_launcher" node that runs the required nodes and launch files on the server side on demand only.  [Here](https://github.com/jrgnicho/ros_examples_metapackage/blob/hydro/rosbridge_example/src/scripts/rosbridge_process_launcher.py) is an example of a potential implementation.  This however will be addressed in the next PR possibly.